rust/sealevel/client/src/main.rs

```diff
@@ -1108,6 +1113,39 @@ fn process_token_cmd(ctx: Context, cmd: TokenCmd) {
                 )
                 .send_with_payer();
         }
+        TokenSubCmd::Igp(args) => match args.cmd {
+            GetSetCmd::Set(set_args) => {
+                let igp_type: InterchainGasPaymasterType = match set_args.igp_type {
+                    IgpType::Igp => InterchainGasPaymasterType::Igp(set_args.igp_account),
+                    IgpType::OverheadIgp => {
+                        InterchainGasPaymasterType::OverheadIgp(set_args.igp_account)
+                    }
+                };
+                let instruction = hyperlane_sealevel_token_lib::instruction::set_igp_instruction(
+                    args.program_id,
+                    ctx.payer_pubkey,
+                    set_args.igp_program,
+                    igp_type,
+                )
+                .unwrap();
+
+                ctx.new_txn().add(instruction).send_with_payer();
```

Comment:
Could you use `add_with_description`? I recently added this so that multisig txs will have some context with them and was hoping for us to slowly move over to using it